nncf/experimental/torch/fx/transformations.py

```diff
@@ -177,8 +186,10 @@ def constant_update_fn(model: torch.fx.GraphModule, node: torch.fx.Node, value:
     new_constant.meta = copy(previous_const.meta)
     new_constant.meta["val"] = value
 
+    consumer_nodes = list(previous_const.users.keys())
     args[input_port_id] = new_constant
-    node.args = tuple(args)
+    for node in consumer_nodes:
```

Comment:
```suggestion
    for node in list(previous_const.users):
```

Reply:
I have removed this code and used `replace_all_uses_with()` instead